spec/takeaway_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:menu) { double :menu }
+  before(:each) { allow(menu).to receive(:content).and_return({ chicken: 4, lamb: 7}) }
```

Comment:
Line is too long. [87/80]<br>Redundant curly braces around a hash parameter.<br>Space inside } missing.